qemu/qmp.go

```diff
@@ -755,6 +755,21 @@ func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevType, netdevID, ifname
 	return q.executeCommand(ctx, "netdev_add", args, nil)
 }
 
+// ExecuteNetdevAddByFds adds a Net device to a QEMU instance
+// using the netdev_add command by fds and vhostfds. netdevID is the id of the device to add.
+// Must be valid QMP identifier.
+func (q *QMP) ExecuteNetdevAddByFds(ctx context.Context, netdevType, netdevID, tapFds, vhostFds string) error {
+	args := map[string]interface{}{
+		"type":     netdevType,
+		"id":       netdevID,
+		"fds":      tapFds,
+		"vhost":    "on",
+		"vhostfds": vhostFds,
```

Comment:
does this device support `disable-modern` ?

Reply:
No. I have tested.